proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -168,52 +174,7 @@ Due to the complexity of the flow, the developer must do the implementation care
 This proposal requires a major version change.
 Since new parameters are added, Core, iOS, Java Suite, RPC, Protocol, and HMI are affected.
 
-
 ## Alternatives considered
```

Comment:
The Steering Committee's requested revisions did not include removing this section.  Even though we will be using the flow described in the `Proposed solution` section, the flow described here is still an alternative that was considered, so should be kept for visibility.  Please add back lines 174 - 219.

Reply:
Hi, @theresalech -san.
Thank you for your review.
I will sdd back lines 174-219 and make a commit it.
Could you please confirm it.
Thank you.

Reply:
@Akihiro-Miyazaki, this does not yet appear to be resolved.

Reply:
Hi, @theresalech -san.
I am very sorry for delay my response.
I modified the proposal documentation and added the image file.
Also, I made a commit them.
Could you please confirm them again.
Thank you.